core/src/main/java/org/mapfish/print/map/style/SLDParserPlugin.java

```diff
@@ -178,21 +178,21 @@ public URL locateResource(final String uri) {
      */
     public static class ErrorHandler extends DefaultHandler {
 
-        private static final Logger LOGGER = LoggerFactory.getLogger(StyleParser.class);
+        private static final Logger LOGGER = LoggerFactory.getLogger(ErrorHandler.class);
 
         /**
          * @param e Exception
          */
         public final void error(final SAXParseException e) throws SAXException {
-            LOGGER.debug(e.getLocalizedMessage());
+            LOGGER.debug("XML error: " + e.getLocalizedMessage());
```

Comment:
Is there a reason to log at `debug` level here and below?

Reply:
Yes, this stupid print is trying to load the style in the following order:
* as an URL
* as an SLD (XML)
* as JSON

Without trying to be smart at all. I've tried to add some code to make it smarter, but that broke so many tests that I've quit. spent too much time on this task already...